src/GOWIN.py

```diff
@@ -163,21 +161,17 @@ def SYN_AND_REPORT_TIMING_NEW(
 
     if not os.path.exists(output_directory):
         os.makedirs(output_directory)
-    log_path = output_directory + "/" + "impl/pnr/" + log_file_name
+    log_path = output_directory + "/impl/pnr/pipelinec.tr"
```

Comment:
Can you change things to make the output log file be `<top_entity_name>.tr`?
(as opposed to 'pipelinec' from -output_base_name flag iiuc)

Is because multiple 'top' versions (diff entity names) all end up in the same output dir and the log files need to not get overridden.

Reply:
So, the log path is generated from `-output_base_name` - I can change that instead.

Reply:
Forgot to add that in the commit message, but the most recent CL in the PR now does that correctly.